test/behavior/vector.zig

```diff
@@ -372,6 +372,7 @@ test "load vector elements via comptime index" {
     if (builtin.zig_backend == .stage2_aarch64) return error.SkipZigTest; // TODO
     if (builtin.zig_backend == .stage2_arm) return error.SkipZigTest; // TODO
     if (builtin.zig_backend == .stage2_sparc64) return error.SkipZigTest; // TODO
+    if (builtin.zig_backend == .stage2_spirv64) return error.SkipZigTest;
```

Comment:
I'm not sure that this should have regressed

Reply:
fails here: https://github.com/ziglang/zig/blob/483183b3bc75d862bcb039a2bf111cc8c8886cac/test/behavior/vector.zig#L382

something like `try expect((&v[2]).* == 3);` works tho. so i assume rusticl doesn't like generic pointers being passed as function parameters?

Reply:
Hmm thats weird, but I guess we can resolve it another time.